requirements.txt

```diff
@@ -3,5 +3,4 @@ pandas==1.3.3
 python-dotenv==0.14.0
 mypy==0.910
 gspread==4.0.1
-data-science-types==0.2.23
```

Comment:
I realized that this project is not supported any more and it is not helping mypy with pandas typing. So I removed it and added more ignores in mypy. 

I did not wanted to write my own types for it